configs/desktop_savant_events_schemas.json

```diff
@@ -0,0 +1,1122 @@
+{
+  "source": "telemetry-cohorts",
+  "filters": {
+    "docType": [
+      "main"
+    ],
+    "appName": [
+      "Firefox"
+    ],
+    "experimentId": [
+      "pref-flip-savant-1457226-de-existing-users",
+      "pref-flip-savant-1457226-de-new-users",
+      "pref-flip-savant-1457226-en-existing-users",
+      "pref-flip-savant-1457226-en-new-users"
+    ]
```

Comment:
This file is copied directly from https://github.com/mozilla/SAVANT-study-analysis/blob/98cfa95e0141e12595c08ea96932a95885368e11/savantConfigFile.json

The only modifications I've made are to add the "source" field at the top and added this filter on experimentId.

Reply:
More modifications have been made, so the above is no longer really true.